plugins/user_quota/server/quota.py

```diff
@@ -279,8 +271,11 @@ def _getBaseResource(self, model, resource):
                  type, either 'user' or 'collection'., and 'resource' is the
                  base resource document or the id of that document.
         """
-        if isinstance(resource, tuple(list(six.string_types) + [ObjectId])):
-            resource = self.model(model).load(id=resource, force=True)
+        if isinstance(resource, six.string_types + (ObjectId,)):
```

Comment:
I'd like to see more strongly typed parameters here; i.e. `resource` should just always be a document, and it should be the responsibility of the caller to load it if necessary.

However, that's out of scope of this PR.